.huskyrc

```diff
@@ -1,3 +0,0 @@
-"hooks": {
-    "pre-commit": "lint-staged"
```

Comment:
`lint-staged` is actually another library that makes the linter only run on "staged" files. I need to get that to work again too, alongside `Prettier` 🤔